### PR TITLE
[Apcu] Load APCUIterator only when APCIterator exists

### LINKED ISSUE #39: APCU polyfill register a class to be autoloadable without taking into account dependency
The \APCUIterator stubs is register to be autoloadable but it extend a class that may very well not exist.
Therfore calling :
`class_exists('\APCUIterator');` 
will trigger a Fatal Error when \APCIterator isnt available.

For exemple, \Stash\Driver\Apc::isAvailable() rely on such a call.

I think, the stubs should only be know by Composer's autoloader if it's instanciable.

### PR DESCRIPTION
**Pull request by nicolas-grekas**
Fixes #39

### CODE REVIEW

**Comment by stof** [comment]
:+1:

**Comment by fgrandjean** [comment]
:+1:

File: src/Apcu/bootstrap.php [hunk 1]
```diff
@@ -26,3 +26,13 @@ function apcu_inc($key, $step = 1, &$success = false) { return apc_inc($key, $st
     function apcu_sma_info($limited = false) { return apc_sma_info($limited); }
     function apcu_store($key, $var = null, $ttl = 0) { return apc_store($key, $var, $ttl); }
 }
+
+if (!class_exists('APCUIterator', false)) {
```
Comment (stof) [inline src/Apcu/bootstrap.php]: Actually, the condition should also check for APCIterator

Comment (nicolas-grekas) [inline src/Apcu/bootstrap.php]: I though that the existing check in https://github.com/symfony/polyfill/pull/40/files#diff-af8f2afac0a67b224b25386b2a75cc16R12 would be enough.
I don't get why, but it looks like so... hhvm?

Comment (keradus) [inline src/Apcu/bootstrap.php]: also, `APCIterator` is available from `PECL apc >= 3.1.1`, so it doesn't exist on 3.0 for example

**Pull request merged**